trunner/harness/plo.py

```diff
@@ -75,7 +76,12 @@ def enter_bootloader(self):
         try:
             self.dut.expect_exact("Waiting for input", timeout=3)
         except (pexpect.TIMEOUT, pexpect.EOF) as e:
-            raise PloError("Failed to go into bootloader mode", output=self.dut.before) from e
+            raise PloError(
+                "Failed to go into bootloader mode", output=self.dut.before
+            ) from e
+
+        # Temporary fix for [https://github.com/phoenix-rtos/phoenix-rtos-project/issues/1040]
+        time.sleep(0.2)
```

Comment:
I asked whether the 0.1 sleep is sufficient - I know that you have increased it to be sure, but IMHO if it worked fine we should leave 0.1 - we don't like sleeps, especially longer ones.